lib/tasks/evm.rake

```diff
@@ -3,6 +3,22 @@ require 'evm_application'
 require 'evm_rake_helper'
 
 namespace :evm do
+  namespace :foreman do
+    task :start => [:environment, 'db:seed'] do
+      server = MiqServer.my_server
+
+      # Assign and activate the default roles
+      server.ensure_default_roles
+      server.activate_roles(server.server_role_names)
+
+      # Mark the server as started
+      server.update_attributes(:status => "started")
```

Comment:
This is going to cause some weirdness with `rake evm:status`, and switching between `rake evm:foreman:start` and then using `rake evm:stop`.  Not that this is a common usecase, but it does cause `rake evm:stop` to basically hang since there is nothing for it to actually stop.